internal/runbits/requirements/requirements.go

```diff
@@ -90,19 +90,23 @@ func ExecuteRequirementOperation(prime primeable, requirementName, requirementVe
 		}()
 	} else {
 		language, err := model.LanguageByCommit(pj.CommitUUID())
-		if err != nil {
-			return locale.WrapError(err, "err_package_get_language", "Could not get language from project")
+		if err == nil {
+			langName = language.Name
+			if nsType == model.NamespacePackage || nsType == model.NamespaceBundle {
+				ns = model.NewNamespacePkgOrBundle(langName, nsType)
+			}
+		} else {
+			logging.Error("Could not get language from project: %v", err)
```

Comment:
```suggestion
			logging.Debug("Could not get language from project: %v", err)
```

This will happen by design for certain use-cases (eg. adding packages when no language has been defined yet).